### PR TITLE
[Feature] Add TS example with NPM/Yarn support

### PR DESCRIPTION
**Pull request by brapoprod**
This PR duplicates and ports the existing basic vanilla JS application to TypeScript and uses the SDK with the help of NPM (or Yarn)

### CODE REVIEW

File: ts-npm-example/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Basic TypeScript and NPM example of SDK usage
+This basic axample shows how simple it is to make a TypeScript integration, as well as using the SDK via NPM instead of a script.
```
Comment (alexandraFlavia9) [inline ts-npm-example/README.md:2]: example

Comment (brapoprod) [reply ts-npm-example/README.md]: Thanks, done in [2443357](https://github.com/chili-publish/editor-sdk-integration-examples/pull/3/commits/2443357083cdad76904eeee88cd487d70c21d320)

File: README.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
 # Editor SDK Integration Examples
 
 In this repository you can find some examples on integrating the editor-sdk.
-We strongly advice to always run your code in a local webserver.
+We strongly advice to always run your code in a local webserver to ensure all funcations are available.
```
Comment (alexandraFlavia9) [inline README.md:4]: functions

**Review by pietervp — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (brapoprod) [reply README.md]: Done in https://github.com/chili-publish/editor-sdk-integration-examples/pull/3/commits/a6b4a00f0e82c5e1d31dbd2e59e14b134390032b

**Review by alexandraFlavia9 — APPROVED** [review]

**Pull request merged**